apps/docs/package.json

```
@@ -11,18 +11,18 @@
     "build:animation": "node scripts/animation-support.js"
   },
   "dependencies": {
-    "@primer/gatsby-theme-doctocat": "^4.6.3",
+    "@primer/gatsby-theme-doctocat": "^5.1.1",
```

Let's pin the version here. We want to do this for all dependencies going forward.

```suggestion
    "@primer/gatsby-theme-doctocat": "5.1.1",
```